astropy/coordinates/builtin_frames/__init__.py

```diff
@@ -37,8 +37,11 @@
 from .cirs import CIRS
 from .itrs import ITRS
 from .hcrs import HCRS
-from .ecliptic import (GeocentricTrueEcliptic, BarycentricTrueEcliptic,
-                       HeliocentricTrueEcliptic, BaseEclipticFrame)
+from .ecliptic import (GeocentricMeanEcliptic, BarycentricMeanEcliptic,
+                       HeliocentricMeanEcliptic,
+                       GeocentricTrueEcliptic, BarycentricTrueEcliptic,
+                       HeliocentricTrueEcliptic,
```

Comment:
I think you need to add all of the new frames here if you want them to appear in the API documentation - right now, `HeliocentricEclipticIAU76` and `CustomBarycentricEcliptic` are missing from [the docs in this PR](https://28908-2081289-gh.circle-artifacts.com/0/home/circleci/project/docs/_build/html/coordinates/index.html)